ecl/eclagent/eclagent.cpp

```diff
@@ -3001,6 +3003,7 @@ char * EclAgent::queryIndexMetaData(char const * lfn, char const * xpath)
                 e->Release();
             else
                 exc.setown(e);
+            failcount++;
         }
     }
 
```

Comment:
Just for me to understand - if the first 2 copies fail but the third is ok then we have a key and it is considered success ?
Probably does not matter but should we go through the parts in a random order ?

Reply:
Yes.  It just needs the header block from any ONE part.  As soon as one part, it breaks (line 2996).

Reply:
Random order sounds good.  Probably worth doing separately.  New ticket relating to this: https://track.hpccsystems.com/browse/HPCC-21703